packages/core/src/js/accordion/Accordion.js

```diff
@@ -55,12 +59,16 @@ class Accordion {
    * @property {string} accordionItemToggles  - The query selector for accordion toggles.
    * @property {string} accordionItemHeaders  - The query selector for accordion headers.
    * @property {string} accordionItemContents - The query selector for accordion contents.
+   * @property {string} accordionExpands      - The query selector for accordion expand toggles.
+   * @property {string} accordionCollapses      - The query selector for accordion collapse toggles.
    */
   _selectors = {
     accordionItems: "",
     accordionItemToggles: "",
     accordionItemHeaders: "",
     accordionItemContents: "",
+    accordionExpands: "",
+    accordionCollapses: "",
```

Comment:
This would follow the same thing as the _dom.

You would have a selector for generic controls, and then selectors for expand/collapse.